.ci_support/linux_64_libevent2.1.10python3.10.____cpython.yaml

```diff
@@ -26,8 +26,6 @@ libevent:
 - 2.1.10
 lz4_c:
 - 1.9.3
-msgpack_c:
-- '6'
```

Comment:
@jjerphan any idea why this has changed?

Reply:
`msgpack_c` and `msgpack_cxx` aren't globally pinned now.

See https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/4854.